### ISSUE TITLE
Metric graph not showing up

### ISSUE THREAD

**Issue by matejkramny** [CachetHQ/Cachet#2361]
I installed with the docker repository, using `docker-compose`.

For some reason there is no graph on the homepage, even when the database contains data points.

Is the docker container missing something that is supposed to trigger cachet to aggregate the data points?

**Comment by jbrooksuk** [comment]
You may need to check the **Display graphs on status page?** setting under Application Setup.

**Comment by matejkramny** [comment]
🤦‍♂️

**Comment by askxiaodong** [comment]
I have the same problem. 

I installed the system from github, version 2.3.10.

mysql> select * from metric_points;
+----+-----------+---------+---------+---------------------+---------------------+
| id | metric_id | value   | counter | created_at          | updated_at          |
+----+-----------+---------+---------+---------------------+---------------------+
|  1 |         1 | 101.000 |       1 | 2017-04-04 16:00:00 | 2017-04-06 07:27:36 |
|  2 |         1 | 101.000 |       1 | 2017-04-04 17:00:00 | 2017-04-06 07:28:19 |
|  3 |         1 | 101.000 |       1 | 2017-04-05 17:00:00 | 2017-04-06 07:36:06 |
+----+-----------+---------+---------+---------------------+---------------------+
3 rows in set (0.00 sec)


mysql> mysql> select * from metrics;
+----+------+--------+-------------+---------------+-----------+---------------+--------+--------------+-----------+-------+---------------------+---------------------+
| id | name | suffix | description | default_value | calc_type | display_chart | places | default_view | threshold | order | created_at          | updated_at          |
+----+------+--------+-------------+---------------+-----------+---------------+--------+--------------+-----------+-------+---------------------+---------------------+
|  1 | test | cup    | test        |         0.000 |         0 |             1 |      1 |            2 |        30 |     0 | 2017-04-06 07:24:34 | 2017-04-06 07:42:07 |
+----+------+--------+-------------+---------------+-----------+---------------+--------+--------------+-----------+-------+---------------------+---------------------+


But I can't find the chart in the status page.

**Comment by jbrooksuk** [comment]
@askxiaodong did you follow the same instructions as https://github.com/CachetHQ/Cachet/issues/2361#issuecomment-277776264?

**Comment by zine2hamster** [comment]
Having the same problem. I tried both 2.3.10 and 2.3.12 in docker-compose. A status 500 is received according to the browser.

![screenshot](http://i.imgur.com/iujbKQz.png)

This is from a freshly installed cachet instance and a newly created metric. Even without the POSTing points, demo.cachethq.io shows that there should be default values of zero. My instance does not show those zeros.

```
cachet_1    | 172.18.0.1 - - [13/Jul/2017:09:06:39 +0000] "GET /dashboard/settings/setup HTTP/1.0" 200 12534 "https://status.stg.kalibrr.com/dashboard/settings/setup" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36" "-"
cachet_1    | 172.18.0.1 - - [13/Jul/2017:09:06:42 +0000] "GET / HTTP/1.0" 200 13804 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36" "-"
postgres_1  | ERROR:  relation "metric_points" does not exist at character 90
postgres_1  | STATEMENT:  select avg(metric_points.value * metric_points.counter) as value FROM chq_metrics m JOIN metric_points ON metric_points.metric_id = m.id WHERE metric_points.metric_id = $1 AND to_char(metric_points.created_at, 'YYYYMMDDHH24') = $2 GROUP BY to_char(metric_points.created_at, 'H')
cachet_1    | 172.18.0.1 - - [13/Jul/2017:09:06:43 +0000] "GET /metrics/1?filter=today HTTP/1.0" 500 3179 "https://status.stg.kalibrr.com/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36" "-"
postgres_1  | ERROR:  relation "metric_points" does not exist at character 90
postgres_1  | STATEMENT:  select avg(metric_points.value * metric_points.counter) as value FROM chq_metrics m JOIN metric_points ON metric_points.metric_id = m.id WHERE metric_points.metric_id = $1 AND to_char(metric_points.created_at, 'YYYYMMDDHH24') = $2 GROUP BY to_char(metric_points.created_at, 'H')
cachet_1    | 172.18.0.1 - - [13/Jul/2017:09:06:47 +0000] "GET /metrics/1?filter=today HTTP/1.0" 500 3179 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36" "-"
cachet_1    | 172.18.0.1 - - [13/Jul/2017:09:18:29 +0000] "GET / HTTP/1.0" 200 13954 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36" "-"
cachet_1    | 2017/07/13 09:18:29 [crit] 112#112: *49 connect() to [::1]:9000 failed (99: Address not available) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /metrics/1?filter=today HTTP/1.0", upstream: "fastcgi://[::1]:9000", host: "status.stg.kalibrr.com", referrer: "https://status.stg.kalibrr.com/"
cachet_1    | 2017/07/13 09:18:29 [warn] 112#112: *49 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /metrics/1?filter=today HTTP/1.0", upstream: "fastcgi://[::1]:9000", host: "status.stg.kalibrr.com", referrer: "https://status.stg.kalibrr.com/"
postgres_1  | ERROR:  relation "metric_points" does not exist at character 90
postgres_1  | STATEMENT:  select avg(metric_points.value * metric_points.counter) as value FROM chq_metrics m JOIN metric_points ON metric_points.metric_id = m.id WHERE metric_points.metric_id = $1 AND to_char(metric_points.created_at, 'YYYYMMDDHH24') = $2 GROUP BY to_char(metric_points.created_at, 'H')
cachet_1    | 172.18.0.1 - - [13/Jul/2017:09:18:29 +0000] "GET /metrics/1?filter=today HTTP/1.0" 500 3179 "https://status.stg.kalibrr.com/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36" "-"
```

**Comment by jbrooksuk** [comment]
Cachet v2.3.* didn't support metrics with prefixed table names... This is fixed in v2.4 though. Might be worth backporting it.

**Comment by bryanbjy** [comment]
Same problem, shouldn't https://github.com/CachetHQ/Cachet/blob/b431ee3702831df88a669c1909cba02d863b4cef/resources/assets/js/components/status-page/Metric.vue#L95 have 'api/v1/metrics' instead of just 'metrics' ? Since all the apis are configured with '/api/v1' suffix.
